perceval/simulators/simulator.py

```diff
@@ -398,3 +396,17 @@ def evolve(self, input_state: Union[BasicState, StateVector]) -> StateVector:
 
         result_sv.normalize()
         return result_sv
+
+    def evolve_svd(self, svd: Union[SVDistribution, StateVector, BasicState]) -> SVDistribution:
+        """Compute the SVDistribution evolved through a Linear Optical circuit"""
```

Comment:
Could you write a complete docstring for this method (you can check other methods of this class)? Actually, the code reference of this class is autogenerated as user documentation.